internal/venice-test-common/src/integrationtest/java/com/linkedin/venice/router/ReplicaFailoverTest.java

```diff
@@ -4,12 +4,12 @@
 import com.linkedin.venice.client.store.AvroGenericStoreClient;
 import com.linkedin.venice.client.store.ClientConfig;
 import com.linkedin.venice.client.store.ClientFactory;
-import com.linkedin.venice.integration.utils.ServiceFactory;
-import com.linkedin.venice.integration.utils.VeniceClusterWrapper;
-import com.linkedin.venice.integration.utils.VeniceRouterWrapper;
-import com.linkedin.venice.integration.utils.VeniceServerWrapper;
 import com.linkedin.venice.listener.request.RouterRequest;
 import com.linkedin.venice.listener.response.HttpShortcutResponse;
+import com.linkedin.venice.services.ServiceFactory;
```

Comment:
We do not want all of these to move into the unit test spaces. These were intentionally isolated to only be allowed to be used by the integration tests

Reply:
But if even we keep it isolated there is no guarantee that it won't be used in UTs. 

Besides we can rename this module to integration tests or something like that?

Reply:
We control the guarantee of it not being used in UTs via gradle module dependencies

Reply:
Oh ok. Reverting it back. Thanks for the context.

Reply:
Also, to answer your other question about why does this help - it helps because this, along with the coverage checkers, forces us to increase coverage via unit tests, rather than relying on integration tests

Reply:
I still don't get why and how moving the service creation classes can help with CC.

Reply:
Because if they are in the same module, nothing is stopping anyone from creating a `VeniceClusterWrapper` and using that to test code. The problem with this is that you cannot guarantee which code paths get scanned and tested. The coverage results could vary quite a bit across runs

Without these, you're forced to write mocks, and thus, have deterministic unit tests

Reply:
Oh okay. Thanks!